pkg/utils/k8sutil.go

```diff
@@ -747,3 +759,33 @@ func addInitContainerAnnotation(dpm *v1beta1.Deployment) error {
 	}
 	return nil
 }
+
+func getKV(label string) (string, string) {
```

Comment:
unittest? documentation?

At the moment I have to reverse-engineer the input syntax for this function - and if it does something different, I have no way of knowing if that's a bug or just my misunderstanding.